### PR TITLE
Added server routes for classes and units

### PR DESCRIPTION
**Pull request by rrrliu**
Fixes everything

### CODE REVIEW

File: client/src/components/Dashboard/ClassBox/ClassBox.jsx [hunk 1]
```diff
@@ -12,8 +12,9 @@ class ClassBox extends Component {
 
   render() {
     const { color, title, teacher } = this.props;
+    const route = `/units/`;
     return (
```
Comment (bchee) [inline client/src/components/Dashboard/ClassBox/ClassBox.jsx]: put a new line here

File: client/src/components/Dashboard/Dashboard.jsx [hunk 2]
```diff
@@ -3,38 +3,50 @@ import ClassBox from './ClassBox/ClassBox';
...
-  'One Weird Ass Class Name Here',
-  'Another Weird Ass Class Name Here',
-  'One Last Weird Ass Class Name Here',
-  'JUST KIDDING ANUTHA ONE YEET',
-  'ANUTHA ONE ANUTHA ONE ANUTHA ONE ANUTHA ONE'
-];
-
-const teachers = ['Teacher', 'Teacher', 'Teacher', 'Teacher', 'Teacher'];
-const colors = ['yellow', 'aqua', 'dark-teal'];
+// const colors = ['yellow', 'aqua', 'dark-teal'];
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: delete this

File: client/src/components/Dashboard/Dashboard.jsx [hunk 3]
```diff
@@ -3,38 +3,50 @@ import ClassBox from './ClassBox/ClassBox';
...
 const topBar = 'top-bar';
 
-function displayClassBoxes(classesArr, teachersArr) {
+function displayClassBoxes(classList) {
   const boxArray = [];
-  for (let i = 0; i < classesArr.length; i += 1) {
-    const color = colors[2 - Math.floor(Math.random() * 3)];
-    const str = `${color} ${topBar}`;
+  for (let i = 0; i < classList.length; i += 1) {
+    // const color = colors[2 - Math.floor(Math.random() * 3)];
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: delete this.

File: client/src/components/Dashboard/Dashboard.jsx [hunk 4]
```diff
@@ -3,38 +3,50 @@ import ClassBox from './ClassBox/ClassBox';
...
     );
   }
   return boxArray;
 }
 
 class Dashboard extends Component {
+  constructor() {
+    super();
+    this.state = {
+      classList: ['hello']
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: why is 'hello' initialized in here

File: client/src/components/Dashboard/Dashboard.jsx [hunk 5]
```diff
@@ -3,38 +3,50 @@ import ClassBox from './ClassBox/ClassBox';
...
+    };
+  }
+
+  async componentWillMount() {
+    const res = await fetch('/api/classes');
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: name this something better

File: client/src/components/Dashboard/Dashboard.jsx [hunk 6]
```diff
@@ -3,38 +3,50 @@ import ClassBox from './ClassBox/ClassBox';
...
+    const json = await res.json();
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: this too

File: client/src/components/Units/Units.jsx [hunk 7]
```diff
@@ -3,30 +3,44 @@ import { NavLink } from 'react-router-dom';
...
-const unitBoxes = [];
-const unitNames = [
-  'House on Mango Street',
-  'Macbeth',
-  "Harry Potter and Sahana's Eyebrow",
-  "Harry Potter and the Gas Smell in Abhi's Apartment",
-  "Harry Potter and Lang's Third Nipple"
-];
-function create() {
+// const url = window.location.href;
```
Comment (bchee) [inline client/src/components/Units/Units.jsx]: delete all these commented out stuff

**Comment by rrrliu** [comment]
submitted my new push

File: client/src/components/Units/Units.jsx [hunk 8]
```diff
@@ -3,30 +3,41 @@ import { NavLink } from 'react-router-dom';
...
   }
+  return unitBoxes;
 }
 
-create();
 class Units extends Component {
+  constructor() {
+    super();
+    this.state = {
+      unitList: [{ name: 'lmao' }]
```
Comment (bchee) [inline client/src/components/Units/Units.jsx]: make unitList empty

File: client/src/components/Dashboard/Dashboard.jsx [hunk 9]
```diff
@@ -3,38 +3,48 @@ import ClassBox from './ClassBox/ClassBox';
...
+function displayClassBoxes(classList) {
   const boxArray = [];
-  for (let i = 0; i < classesArr.length; i += 1) {
-    const color = colors[2 - Math.floor(Math.random() * 3)];
-    const str = `${color} ${topBar}`;
+  for (let i = 0; i < classList.length; i += 1) {
+    const str = `${classList[i].color} ${topBar}`;
     boxArray.push(
-      <ClassBox title={classesArr[i]} teacher={teachersArr[i]} color={str} />
+      <ClassBox
```
Comment (bchee) [inline client/src/components/Dashboard/Dashboard.jsx]: this needs a key prop set it equal to the same as id

**Pull request merged**